src/components/paragraph/stanford-lists.tsx

```diff
@@ -140,6 +144,12 @@ const getViewPagedItems = cache(async (viewId: string, displayId: string, contex
       items = graphqlResponse.stanfordSharedTags?.results as unknown as NodeUnion[]
       break
 
+    case 'sul_shared_tag_events--card_grid':
```

Comment:
I'm not sure if this is necessary or not.

Reply:
yes it definitely is needed.